migrator.go

```diff
@@ -694,3 +702,30 @@ func groupByIndexName(indexList []*Index) map[string][]*Index {
 func (m Migrator) GetTypeAliases(databaseTypeName string) []string {
 	return typeAliasMap[databaseTypeName]
 }
+
+// should reset prepared stmts when table changed
+func (m Migrator) resetPreparedStmts() {
+	if m.DB.PrepareStmt {
+		if pdb, ok := m.DB.ConnPool.(*gorm.PreparedStmtDB); ok {
+			pdb.Reset()
+		}
+	}
+}
+
+func (m Migrator) DropColumn(dst interface{}, field string) error {
```

Comment:
Seems like `DropColumn`, `RenameColumn` is not necessary to pass current test cases

Reply:
I added test cases for them